delivery_recipe/Delivery/README.md

```diff
@@ -1,2 +1,26 @@
 # Delivery
 
+Our delivery process is designed to handle any and all types of project. Rather than being tailored to a specific type of project our processes focus on what’s important in any project:
+
+* Business goals - what is the business seeking to achieve?
+* Users - who are your users and what are their real needs?
+
+Once a project is agreed and the initial discovery process is complete the high level requirements will be captured as epics. It’s here that the delivery team pick up the process.
+
+## Project Kickoff
+
+As a customer, your first encounter with the project delivery team will be during the Project Kickoff. During this stage our teams will get to know each other and we’ll begin the process of diving into the detail of the requirements, discussing the possible approaches, how we’ll manage the risks and how we’ll exploit the opportunities. At the end of the kickoff we’ll have agreed the initial priority requirements that we’ll take into the first development cycle.
```

Comment:
Won't their first encounter be during the sales stage, or at latest during discovery?